src/app/admin/admin-import-metadata-page/metadata-import-page.component.html

```diff
@@ -1,15 +1,21 @@
 <div class="container">
   <h2 id="header">{{'admin.metadata-import.page.header' | translate}}</h2>
   <p>{{'admin.metadata-import.page.help' | translate}}</p>
+  <p>
+    <input type="checkbox" [(ngModel)]="validateOnly" />
+    <span class="ml-1">{{'admin.metadata-import.page.validateOnly' | translate}}</span>
```

Comment:
Can we add a hint directly under this checkbox which says:
```
When selected, the uploaded CSV will be validated. You will receive a report of detected changes, but no changes will be saved.
```

Reply:
done